docs/actions.md

```diff
@@ -202,6 +202,39 @@ Parameters can be passed to the action when it is invoked.
       "payload": "Hello, Bernie from Vermont"
   }
   ```
+4.  It is possible to pipe JSON parameters to the `wsk` command by using the special filename `-` with `--params-file` or `-P`:
+  
+  ```
+  cat parameters.json | envsubst | wsk action invoke -r hello -P -
+  ```
+  
+  This example uses `envsubst` to replace environment variables in the JSON before passing them to the action invocation.
```

Comment:
for the sake of the example is `envsubst` necessary? I can see the counter argument it's already in a file why use `-`. How about using echo instead?

```bash
echo '{"name": "Alice"}' | wsk action invoke hello -r -P -
```

Reply:
I thought it's nice to have a more "real world" use case example.

The echo example can also be done simpler using `-p name Alice`. I chose `envsubst` as that's an actual case where you need to pipe from a file since some processing is going on, and I happened to be using `envsubst` in my case.

Do you think there is a need to be as simple as possible?

Reply:
OK - I was going for simpler example but after your explanation it's good to have a real use case.